### PR TITLE
Keep modifiers when implementing abstract members

### LINKED ISSUE #1759: "Implement all members" ignores access modifier
**Describe the bug**
When a concrete class has unimplemented members, the quick fix to "Implement all members" ignores any access modifier, generally `protected` (and instead adds an `override`).

**To Reproduce**
Self explanatory.

**Expected behavior**
Keep the intended access modifier, and preferably not add the semantically inaccurate `override`.

**Additional context**
Personally I dislike having `override` for methods that don't, strictly speaking, override.
But it's a matter of style, so it would be nice to have that as an optional setting.

### PR DESCRIPTION
**Pull request by kpbochenek**
Fixes #1759

Not sure if this is a bug or intended behaviour but `flagString` generates modifiers like `protected[package x.y.z]` and they need to be stripped manually.

### CODE REVIEW

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala [hunk 1]
```diff
@@ -213,6 +213,16 @@ trait OverrideCompletions { this: MetalsGlobal =>
         if (sym.isLazy) "lazy "
         else ""
 
+      val _modifs = sym.flagString
+        .replace("package ", "")
```
Comment (tgodzik) [inline mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]: Do we need to do this? We are not doing overrides for any of those symbol types.

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala [hunk 2]
```diff
@@ -233,6 +243,8 @@ trait OverrideCompletions { this: MetalsGlobal =>
       val name: String = Identifier(sym.name)
 
       val filterText: String = s"${overrideKeyword}${lzy}${keyword}${name}"
+      val insertText: String =
+        s"${overrideKeyword}${modifs}${keyword}${name}$signature"
```
Comment (tgodzik) [inline mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:247]: Will this work if we want to override lazy vals? They cannot be abstract but we can override them and this compilation is also used for non-abstract symbols.

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 1 above)
Comment (kpbochenek) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]:
```
Not sure if this is a bug or intended behaviour but flagString generates modifiers like protected[package x.y.z] and they need to be stripped manually.
```

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 2 above)
Comment (kpbochenek) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:247]: lazy should be in modifs already, I wanted to just get rid of lzy and only use modifs but it is used to match against what user typed and matching against modifiers is probably not what user wants.
Gonna double check lazy is there :+1:

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]: What about `object`, `trait` or `class` ?

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 2 above)
Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:247]: Cool, let's add a test if there is none.

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 1 above)
Comment (kpbochenek) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]: handled already

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 2 above)
Comment (kpbochenek) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:247]: yes, there is already a test :+1:
```
  checkEditLine(
    "lazy",
    """|package q
       |abstract class Val {
       |  lazy val hello: Int = 2
       |}
       |class Main extends Val {
       |   ___
       |}
       |""".stripMargin,
    "override val hel@@",
    "override lazy val hello: Int = ${0:???}"
  )
```

File: mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala (hunk 1 above)
Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]: I meant, does those keywords can appear here even? You mentioned `protected[package x.y.z] `, but why would trait appear here?

Comment (kpbochenek) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]:
```
scala> trait A {
     |   trait AB {
     |     private[A] val x = 3
     |   }
     | }
trait A
```

Comment (tgodzik) [reply mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala:217]: We could do:
```
      val _modifs =
        sym.flagString.replace(
          sym.privateWithin.toString(),
          sym.privateWithin.name.toString()
        )
```
